### ISSUE TITLE
sfdisk -l can fail with return code 1 and error No medium found

### ISSUE THREAD

**Issue by reimannf** [sapcc/swift-drive-autopilot#20]
So far we assumed that `sfdisk -l` on devices which are not readable will print nothing to stdout and return successful, see https://github.com/sapcc/swift-drive-autopilot/blob/master/collectors.go#L137-L141.

Now we have several regions where that behaves different and `sfdisk -l` will fail with exit code 1 and stderr `sfdisk: cannot open <device>: No medium found`.
That causes `swift-drive-autpliot` to exit and crashlooping and the dependent services can not start because of the `drives-ready-flag`.

The intended behaviour to just ignore such devices should also apply to the described failure, so there might be some evaluation of stderr needed.

Example log output
```
2018/02/03 21:11:41 Output from sfdisk: sfdisk: cannot open /dev/sdo: No medium found
2018/02/03 21:11:41 FATAL: exec(chroot /coreos sfdisk -l /dev/sdo) failed: exit status 1
```

**Comment by reimannf** [comment]
It is very likely that the `sfdisk` changed with coreos upgrade:
No exit code: `Container Linux by CoreOS stable (1576.5.0)`
Exit code 1: `Container Linux by CoreOS stable (1632.2.1)`

**Comment by majewsky** [comment]
Fixed by #21.
